editor.planx.uk/src/pages/FlowEditor/components/Settings/ServiceSettings.tsx

```diff
@@ -24,6 +28,78 @@ import InputRowItem from "ui/shared/InputRowItem";
 import type { FlowSettings } from "../../../../types";
 import { useStore } from "../../lib/store";
 
+const CopyButton = (props: { link: string; isActive: boolean }) => {
+  const [copyMessage, setCopyMessage] = useState<"copy" | "copied">("copy");
+  return (
+    <Tooltip title={copyMessage}>
+      <Button
+        disabled={!props.isActive}
+        component={"button"}
+        variant="help"
+        onMouseLeave={() => {
+          setTimeout(() => {
+            setCopyMessage("copy");
+          }, 500);
+        }}
+        onClick={() => {
+          setCopyMessage("copied");
+          navigator.clipboard.writeText(props.link);
+        }}
+        sx={{ marginLeft: 0.5 }}
+      >
+        <ContentCopyIcon style={{ width: "18px", height: "18px" }} />
+        <Typography ml={0.5} variant="body3">
+          {copyMessage}
+        </Typography>
+      </Button>
+    </Tooltip>
+  );
+};
+
+const TitledLink: React.FC<{
+  link: string;
+  isActive: boolean;
+}> = ({ link, isActive }) => {
+  return (
+    <Box paddingBottom={0.5} mt={1}>
+      <Typography mb={1} variant="h4">
+        Your public link
+        <CopyButton isActive={isActive} link={link} />
+      </Typography>
+      {isActive ? (
+        <Link variant="body2" href={link}>
+          {link}
+        </Link>
+      ) : (
+        <Typography style={{ color: "GrayText" }} variant="body2">
+          {link}
+        </Typography>
```

Comment:
These links aren't taking me where I'd expect them to! 

If I set a fake test subdomain, it's trying to _internally_ route me, rather than treating it as an external route - eg 
![Screenshot from 2024-08-16 14-01-41](https://github.com/user-attachments/assets/0cd5460f-ad8b-4bd5-95a8-68312f4ce16e)
is pointing to `https://3525.planx.pizza/newcastle/apply-for-planning-permission/planningservices.test.gov.uk/apply-for-planning-permission`

Also two minor things:
- I'd expect this to open in a new tab by default?
- Can we use default `Link` styles here (eg underline) rather than `variant="body2"` ?

Reply:
Let's also chat about how to add test coverage to catch this if helpful ! Still think this is an important feature that would benefit from a few tests :+1:

Reply:
Ah! yeah - if you don't have a protocol prefix `https://...` it'll just add it on to the current URL.  Would a better way to route it be including ``href={ "//" + link }`` or should it be something similar to the trailing end "/" where it's managed when adding a new `subdomain`?

Added ``target={ _blank ) rel...`` to the Link now. Good catch!

For the last point, you mean for an inactive link, we should add the underline? Or switch the component to a ``<Link>``?

Reply:
Personally I'd add the `https://` direct via a string template - tacking onto the comment above, could be: 
```const subdomainLink = teamDomain && `https://${teamDomain}/${flowSlug}`;``` ? 

A disabled `<Link />` would make semantic sense to me (haven't double checked MUI API docs if this is actually possible) or just underlined to match default PlanX link styles!

Reply:
Sick, so we assume teamDomain never has "https://" and I'll always add it.

Yeah I couldn't see a ``disabled`` prop in the `<Link>` [API](https://mui.com/material-ui/api/link/), so I'll add the underline